File: src/API/WesternStatesWater.WestDaat.Contracts.Client/Responses/Conservation/ReviewerConservationApplicationLoadResponse.cs
namespace WesternStatesWater.WestDaat.Contracts.Client.Responses.Conservation;

public class ReviewerConservationApplicationLoadResponse : ApplicationLoadResponseBase
{
    public Common.DataContracts.ApplicationDetails Application { get; set; } = null!;

    public Common.DataContracts.ApplicationReviewNote[] Notes { get; set; } = null!;
}

File: src/DashboardUI/src/data-contracts/ReviewerConservationApplicationLoadResponse.ts
import { ApplicationDetails } from './ApplicationDetails';
import { ApplicationLoadResponseBase } from './ApplicationLoadResponseBase';
import { ApplicationReviewNote } from './ApplicationReviewNote';
import { ReviewPipeline } from './ReviewPipeline';

export interface ReviewerConservationApplicationLoadResponse extends ApplicationLoadResponseBase {
  application: ApplicationDetails;
  notes: ApplicationReviewNote[];
  reviewPipeline: ReviewPipeline;
}
